### ISSUE TITLE
Not showing notifications for all e-mails

### ISSUE THREAD

**Issue by lPeichl** [Foundry376/Mailspring#1527]
Even though is enabled. Windows 10

**Comment by lPeichl** [comment]
This is important need a fix asap

**Comment by frenzi2012** [comment]
No sound and notifications popup for linux mint 19.1

**Comment by bengotow** [comment]
Hey folks! Will take a look at this, could you check that you're not sending yourself emails? Mailspring doesn't show notifications when you send mail between your own connected accounts (I /think/). Will see if I can reproduce this on a windows install.

**Comment by frenzi2012** [comment]
No, it does not sound with any sender, only the tray icon turns blue, but it does not sound, and there is no system popup notification

**Comment by frenzi2012** [comment]
At some point, only sometimes, when the application is closed but the tray icon is enabled, the sound appears, but when I open the application it no longer sounds.  not even being minimized
